rdmo/management/assets/js/hooks/useBool.js

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react'
 
-const useBool = (inital) => {
-  const [value, setValue] = useState(inital)
+const useBool = (initial) => {
```

Comment:
Is this a typo? Is "initial" a reserved keyword in Javascript? If not, it can be fixed, right?

Reply:
No, its a typo.